torch/ao/quantization/pt2e/utils.py

```diff
@@ -11,12 +11,83 @@
 from torch.utils._pytree import LeafSpec
 from torch._export import capture_pre_autograd_graph
 
+from torch.ao.quantization.quantizer import QuantizationAnnotation
+from torch.ao.quantization.quantizer.utils import _is_sym_size_node
+
 __all__ = [
     "fold_bn_weights_into_conv_node",
     "get_aten_graph_module",
     "remove_tensor_overload_for_qdq_ops",
+    "filter_sym_size_users",
+    "find_q_dq_node_for_user",
+    "is_valid_annotation",
```

Comment:
nit: could you make these private?

Reply:
I did. and having to export it from util was complaining that if this is to be used elsewhere remove `_`. If you have suggestions let me know and i can do it in subsequent diffs

Reply:
I see, maybe I can try to remove, I do remember we have been successfully using private utils somewhere